simplivity/connection.py

```diff
@@ -43,7 +43,7 @@ def __init__(self, ovc_ip, ssl_bundle=False, timeout=None):
         self._password = None
         self._access_token = None
 
-        self._headers = {'Accept': 'application/vnd.simplivity.v1+json'}
+        self._headers = {'Accept': 'application/json'}
```

Comment:
Is this tested with all the resource methods?
The reason why we made application/vnd.simplivity.v1+json as the default value is most of the simplicity REST endpoints accept this type and we don't have to pass this header all the time.

Reply:
It appears that every new feature that is added to the rest api that wasn't in v1+ requires a custom header if we go with the 'Accept': 'application/vnd.simplivity.v1+json' such has 'cluster_groups'.  

I have talked to the rest development team and validated that all the example scripts work with this change, and have tested a get method within all the resources using the swagger UI using the 'Accept': 'application/json' and they work. 

i.e.

curl -X GET --header "Accept: application/json" "https://<IP_Address>/api/backups?limit=500&offset=0&case=sensitive"

curl -X GET --header "Accept: application/json" "https://<IP_Address>/api/cluster_groups?limit=500&offset=0&case=sensitive"

curl -X GET --header "Accept: application/json" "https://<IP_Address>/api/datastores?show_optional_fields=false&limit=500&offset=0&case=sensitive"

curl -X GET --header "Accept: application/json" "https://<IP_Address>/api/external_stores?limit=500&offset=0&case=sensitive"

curl -X GET --header "Accept: application/json" "https://<IP_Address>/api/hosts?show_optional_fields=false&limit=500&offset=0&case=sensitive"

curl -X GET --header "Accept: application/json" "https://<IP_Address>/api/omnistack_clusters?show_optional_fields=false&limit=500&offset=0&case=sensitive"

curl -X GET --header "Accept: application/json" "https://<IP_Address>/api/policies?limit=500&offset=0&case=sensitive"

curl -X GET --header "Accept: application/json" "https://<IP_Address>/api/certificates"

curl -X GET --header "Accept: application/json" "https://<IP_Address>/api/virtual_machines?show_optional_fields=false&limit=500&offset=0&case=sensitive"